terraform/modules/livy-proxy/ecs.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_ecs_task_definition" "livy-proxy" {
+  family                   = var.name
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "256"
+  memory                   = "2048"
+  execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
+
+  container_definitions = <<DEFINITION
+[
+  {
+    "image": "${var.image_ecr_repository}:latest",
+    "name": "squid-s3",
+    "networkMode": "awsvpc",
+    "portMappings": [
+      {
+        "containerPort": ${var.container_port}
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${aws_cloudwatch_log_group.livy-proxy.name}",
+        "awslogs-region": "${data.aws_region.current.name}",
+        "awslogs-stream-prefix": "container-internet-proxy"
```

Comment:
Why does the prefix mention internet-proxy?

Reply:
Yep, my bad